documentation/guide-usecase.asciidoc

```diff
@@ -53,23 +56,34 @@ public interface UcApproveBooking {
 ----
 
 == Implementation
-For the implementation of a use-cas, the same rules that are described for the link:guide-component-facade.asciidoc#implementation[component-facade implementation].
-
- .
-
-However, when following the use-case approach, your component facade simply changes to:
+For the implementation of a use-case, the same rules that are described for the link:guide-component-facade.asciidoc#implementation[component-facade implementation].
```

Comment:
As we do not recommend this concept anymore, I would not like to it here. Maybe we can just drop this sentence as eveything follows in the next one.